packages/math-input/src/components/input/math-input.tsx

```diff
@@ -626,6 +667,33 @@ class MathInput extends React.Component<Props, State> {
         }
     };
 
+    // We want to allow the user to be able to focus the input via click
+    // when using ChromeOS third-party browsers that use mobile user agents,
+    // but don't actually simulate touch events.
+    handleClick: (arg1: React.MouseEvent<HTMLDivElement>) => void = (e) => {
```

Comment:
Is it possible to combine the typing with the declaration? Something like this?

```suggestion
    handleClick: (e: React.MouseEvent<HTMLDivElement>): void {
```